core/chain/chains/evm/tx/getParsedMemo.ts

```
@@ -0,0 +1,68 @@
+            const abi = new Interface([`function ${textSignature}`])
+            const [fragment] = textSignature.split('(')
+
+            try {
+              const decodedData = abi.decodeFunctionData(fragment, memo)
+              const processedData = processDecodedData(decodedData)
+
+              return {
+                functionArguments: JSON.stringify(processedData, null, 2),
+                functionSignature: textSignature,
+              }
+            } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)